repo/fsrepo/fsrepo.go

```diff
@@ -562,7 +562,9 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 
 	var du uint64
 	err = filepath.Walk(pth, func(p string, f os.FileInfo, err error) error {
-		du += uint64(f.Size())
+		if f != nil {
```

Comment:
what i'm saying is that the err in our walkfunc is any error that gets returned from the lstat you link to. I'm saying that we should log that error just in case. I'm always against blanket ignoring errors.